web/app/scripts/views/record/add-edit-partial.html

```diff
@@ -42,7 +42,7 @@
                             </div>
                         </div>
                         <div class="col-md-12">
-                            <div class="map" leaflet-map driver-embed-map style="height:300px;"></div>
+                            <div class="map" leaflet-map driver-embed-map editable="true"></div>
```

Comment:
Would it be hard to add attributes for latitude and longitude? It would make it super easy to add an embedded map without needing to fire an event to update the coords.